File: Servers/JS/Fastify/routers/dbSelect.js
const { QUERY_1, QUERY_1K, QUERY_10K, QUERY_100K } = require("../../db/query");
const { dbSelect } = require("../../utils/paths");
const selectRouter = async (fastify) => {
  fastify.get(dbSelect.oneRecord, async () => {
    const client = await fastify.pg.connect();
    try {
      const { rows } = await client.query(QUERY_1);
      return { len: rows.length };
    } finally {
      client.release();
    }
  });
  fastify.get(dbSelect.oneKRecord, async () => {
    const client = await fastify.pg.connect();
    try {
      const { rows } = await client.query(QUERY_1K);
      return { len: rows.length };
    } finally {
      client.release();
    }
  });
  fastify.get(dbSelect.tenKRecords, async () => {
    const client = await fastify.pg.connect();
    try {
      const { rows } = await client.query(QUERY_10K);
      return { len: rows.length };
    } finally {
      client.release();
    }
  });
  fastify.get(dbSelect.hundredKRecord, async () => {
    const client = await fastify.pg.connect();
    try {
      const { rows } = await client.query(QUERY_100K);
      return { len: rows.length };
    } finally {
      client.release();
    }
  });
};
module.exports = selectRouter;

File: Servers/Python/Flask/routers/db_select.py
from flask import Blueprint

from Servers.Python.utils.db.sync_query.db_con import cursor
from Servers.Python.utils.db.sql_query import *
from Servers.Python.utils.path import DbSelect

bp = Blueprint(
    'db_select_blueprint',
    __name__
)


@bp.route(DbSelect.one_record)
def one_record():
    cursor.execute(
        QUERY_1
    )
    return {'len': len(cursor.fetchall())}


@bp.route(DbSelect.one_k_records)
def one_k_records():
    cursor.execute(
        QUERY_1K
    )
    return {'len': len(cursor.fetchall())}


@bp.route(DbSelect.ten_k_records)
def ten_k_records():
    cursor.execute(
        QUERY_10K
    )
    return {'len': len(cursor.fetchall())}


@bp.route(DbSelect.hundred_k_records)
def hundred_k_records():
    cursor.execute(
        QUERY_100K
    )
    return {'len': len(cursor.fetchall())}
